### PR TITLE
About Page aligned properly

### LINKED ISSUE #2976: About Page not aligned properly for mobile users.
### p5.js version

1.9.0

### What is your operating system?

Windows

### Web browser and version

chrome://121.0.6167.85

### Actual Behavior

The about page is not aligned properly below 641px width , the spacing between them is very less which is not user friendly like this 
Screenshot 2024-01-29 193048

### Expected Behavior

The about should be aligned properly like this , because this is how about page was getting displayed above 641px width.
Screenshot 2024-01-29 193004

### Steps:
1. Go to the project on any browser.
2. Then go to about page. 
3. Then check responsiveness of this page and below 641px width you could see the issue.

### PR DESCRIPTION
**Pull request by rahulrana701**
Fixes #2976

Changes:

I have verified that this pull request:

* [X] has no linting errors (`npm run lint`)
* [X] has no test errors (`npm run test`)
* [X] is from a uniquely-named feature branch and is up to date with the  `develop` branch.
* [X] is descriptively named and links to an issue number, i.e. `Fixes #2976 `

### CODE REVIEW

**Comment by rahulrana701** [comment]
@lindapaiste  I have aligned the about page properly

File: client/styles/components/_about.scss [hunk 1]
```diff
@@ -96,3 +96,9 @@
 .about__footer-list {
   padding-top: #{math.div(12, $base-font-size)}rem;
 }
+
+@media (max-width:641px){
```
Comment (lindapaiste) [inline client/styles/components/_about.scss:104]: 641px is not the right breakpoint.  It's 650px. 
https://github.com/processing/p5.js-web-editor/blob/91e5b79069c7d38555082b6f25bd566b8dbf74af/client/styles/components/_overlay.scss#L32

**Comment by rahulrana701** [comment]
@lindapaiste done as suggested

**Comment by raclim** [comment]
I'm going to close this due to reasons laid out in the associated issue!

**Pull request closed without merging**